ingestors/bookmarks.go

```diff
@@ -3,12 +3,13 @@ package ingestors
 import (
 	"context"
 	"fmt"
+	"time"
 
 	"github.com/librariesio/depper/redis"
 )
 
-// Use to set a date/string bookmark for an ingestor
-func setBookmark(namer Namer, bookmark interface{}) (interface{}, error) {
```

Comment:
This didn't work out, the date was still coming back as a string, so I'm just being safer and relying on concrete functions for times and string, letting the consumer choose what to use for the bookmark, but handling the date twiddling internally.

Reply:
Have you seen any cases yet where we wouldn't use a timestamp?

Reply:
Yea, the sequence number in npm :D